libs/sdk-ui-dashboard/src/model/index.ts

```diff
@@ -37,3 +46,7 @@ export {
     DashboardCommandType,
 } from "./commands/dashboard";
 export { DashboardContext } from "./types/commonTypes";
+
+export { DashboardConfig } from "./types/commonTypes";
```

Comment:
Nitpick: duplicate import (the previous one has the same source file.

Reply:
fixed